### DISCUSSION TITLE
Predicate pushdown in table function APIs?

### DISCUSSION THREAD

**Discussion by j-baker** [duckdb/duckdb#9644] [Ideas]
Hi! A lot of external data formats can be more efficiently queried if properties of the query are known. It seems as though DuckDB has some support for predicate pushdown (e.g. Parquet files observe metadata pruning behaviour), but that this is not exposed to normal table functions.
Are there plans to enhance this over time? My use case is that I'd like to integrate DuckDB with delta lake reading.

**Comment by prashanthellina** [comment]
@j-baker Do you know if there is support for this now?
